docs/commands/push.md

```diff
@@ -69,27 +69,29 @@ To authenticate to the API registry, you can use several approaches:
-| --version        | boolean  | Show version number.                                                                                                                                                                                                                                                                                                                        |
-| --files          | [string] | List of other folders and files to upload. See [the Files section](#files) for more information.                                                                                                                                                                                                                                            |
+| Option           |   Type   | Description                                                                                                                                                                                                                                                     |
+| ---------------- | :------: | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| api              |  string  | The API definition that you want to push to the Redocly API registry. Provide it as a path to the root API definition file (or as an alias). See [Set options explicitly](#set-options-explicitly) for more information.                                        |
+| --destination    |  string  | The location in the API registry where you want to push or upsert your API definition. Provide it in the following format: `api-name@api-version`.                                                                                                              |
+| --organization   |  string  | Specifies the ID of organization that the deffenition is being pushed to. Overrides the one defined in the config file.                                                                                                                                         |
+| --branch, -b     |  string  | The branch where your API definition is pushed or upserted. Default value is `main`.                                                                                                                                                                            |
+| --job-id         |  string  | Specifies the ID of the CI job that the current push is associated with. See [the Batch ID section](#batch-id) for more information.                                                                                                                            |
```

Comment:
Should we be linking to the "Job ID section" now we've renamed this?

Reply:
Yes, thanks